dts/arm/renesas/rcar/gen4/r8a779f0.dtsi

```diff
@@ -10,6 +10,14 @@
 
 / {
 	soc {
+		pfc: pin-controller@e6050000 {
+			compatible = "renesas,rcar-pfc";
+			reg = <0xe6050000 0x16c>, <0xe6050800 0x16c>,
+				  <0xe6051000 0x16c>, <0xe6051800 0x16c>,
+				  <0xdfd90000 0x16c>, <0xdfd90800 0x16c>,
```

Comment:
Add a comment for address conversion over control domain ?

Reply:
epxlained that last four registers were located in control domain of the soc